micro_sam/util.py

```diff
@@ -479,7 +479,7 @@ def get_centers_and_bounding_boxes(
 
 def load_image_data(
     path: str,
-    ndim: int,
+    ndim: Optional[int] = None,
```

Comment:
Is it ok to make this optional? From what I can see it is not used. And for the gui it is difficult to make an assumption on the users selection.

Reply:
It can actually be removed. I forgot that when updating the function. I will remove it after merging the PR.